packages/kg-default-nodes/lib/nodes/html/HtmlNode.js

```diff
@@ -0,0 +1,104 @@
+import {createCommand} from 'lexical';
+import {KoenigDecoratorNode} from '../../KoenigDecoratorNode';
+import {renderHtmlNodeToDOM} from './HtmlRenderer';
+
+export const INSERT_HTML_COMMAND = createCommand();
+
+export class HtmlNode extends KoenigDecoratorNode {
+    __html;
+
+    static getType() {
+        return 'html';
+    }
+
+    static clone(node) {
+        return new this(node.getDataset(), node.__key);
+    }
+
+    // used by `@tryghost/url-utils` to transform URLs contained in the serialized JSON
+    static get urlTransformMap() {
+        return {
+            html: 'html'
+        };
+    }
+
+    getDataset() {
+        const self = this.getLatest();
+        return {
+            html: self.__html
+        };
+    }
+
+    static importJSON(serializedNode) {
+        const {html} = serializedNode;
+        const node = new this({html});
+        return node;
+    }
+
+    exportJSON() {
+        return {
+            type: 'html',
+            version: 1,
+            html: this.getHtml()
+        };
+    }
+
+    constructor({html} = {}, key) {
+        super(key);
+        this.__html = html;
+    }
+
+    exportDOM(options = {}) {
+        const element = renderHtmlNodeToDOM(this, options);
+        return {
+            element,
+            type: 'inner'
+        };
+    }
+
+    /* c8 ignore start */
+    createDOM() {
+        const element = document.createElement('div');
+        return element;
+    }
+
+    updateDOM() {
+        return false;
+    }
+
+    isInline() {
+        return false;
+    }
+    /* c8 ignore stop */
+
+    getHtml() {
+        return this.__html;
```

Comment:
This is missing the `self = this.getLatest()` call meaning we could have out-of-sync issues when the value is synced with the React components.